airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/formatter/GcsCsvBigQueryRecordFormatter.java

```diff
@@ -13,8 +17,22 @@
  */
 public class GcsCsvBigQueryRecordFormatter extends DefaultBigQueryRecordFormatter {
 
+  public static final com.google.cloud.bigquery.Schema CSV_SCHEMA = com.google.cloud.bigquery.Schema.of(
+      Field.of(JavaBaseConstants.COLUMN_NAME_AB_ID, StandardSQLTypeName.STRING),
+      Field.of(JavaBaseConstants.COLUMN_NAME_DATA, StandardSQLTypeName.STRING),
+      Field.of(JavaBaseConstants.COLUMN_NAME_EMITTED_AT, StandardSQLTypeName.TIMESTAMP));
+
   public GcsCsvBigQueryRecordFormatter(JsonNode jsonSchema, StandardNameTransformer namingResolver) {
     super(jsonSchema, namingResolver);
   }
 
+  @Override
+  public Schema getBigQuerySchema(JsonNode jsonSchema) {
+    // if (TypingAndDedupingFlag.isDestinationV2()) {
```

Comment:
bigquery 2.x is always on v2, so you can delete the commented-out bits

Reply:
Done